### PR TITLE
CSRF token issue and mod button styling fix

### PR DESCRIPTION
**Pull request by nsixcancode**
Also reorganized the house statistics so user avg and total time are on top of each other.

### CODE REVIEW

File: app/controllers/houses_controller.rb [hunk 1]
```diff
@@ -26,11 +26,10 @@ def update
...
     old_house_name = @house.house_name
     respond_to do |format|
-      if @house.update(house_params) && mod_params[:reason].present?
+      if @house.update(house_params)
         log_update(old_house_name)
         format.html { redirect_to '/houses' }
       else
-        @house.errors.add(:base, 'Must specify a reason.') if mod_params[:reason].blank?
         format.html { render :edit }
       end
```
Comment (jamestmartin) [inline app/controllers/houses_controller.rb:34]: Respond to ... format is redundant if the only format you support is HTML. You can remove remove the `respond_to` and `format.html` blocks entirely and just put the `render` directly in the method body.

File: app/controllers/submissions_controller.rb [hunk 2]
```diff
@@ -39,7 +39,7 @@ def new
     @participations = Participation.where({ user_id: current_user.id, active: true }).order('challenge_id ASC')
     # Inform the user of their houses score (if they are participating)
     @house_participations = current_user.house_participations.where("join_date >=  ?", Time.now.utc.at_beginning_of_month.to_date).first
-    @points = @house_participations ? @house_participations.score : 0
+    @time = @house_participations ? @house_participations.score : 0
```
Comment (jamestmartin) [inline app/controllers/submissions_controller.rb:42]: Not necessarily better, but there's also the option of `@house_participations&.score || 0`, if you prefer that syntax. (I think that'd work anyway, I haven't tried it.)

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -64,11 +64,12 @@ def create
   end
 
   def edit
-    @followers = Follower.where({ user: params[:id]}).includes(:following)
+    @followers = Follower.where({user: params[:id]}).includes(:following)
```
Comment (jamestmartin) [inline app/controllers/users_controller.rb:67]: The curly brackets are redundant, try `Follower.where(user: params[:id])`.

File: app/views/houses/edit.html.slim [hunk 4]
```diff
@@ -15,7 +15,7 @@
           .col-9 = h.text_field :house_name
         .form-group.row
           .col-3 = label :reason, "Reason:"
-          .col-9 = text_field :reason, :reason
+          .col-9 = text_field :reason, :reason, required: true
```
Comment (jamestmartin) [inline app/views/houses/edit.html.slim:18]: In the controller you made it *no longer* required, but you're making it required in the form?

File: app/views/submissions/_form.html.slim [hunk 5]
```diff
@@ -210,13 +210,13 @@ javascript:
         .card-body
           - if @house_participations
             p.card-text
-              | You have earned #{@points.to_s} #{"point" + (@points == 1 ? " " : "s ")} for your house this month
+              | You have earned #{@time / 60} hr and #{@time % 60} #{"min" + (@time % 60 == 1 ? " " : "s ")} for your house this month
```
Comment (jamestmartin) [inline app/views/submissions/_form.html.slim:213]: You pluralized mins but not hrs. Might be intentional.

**Review by jamestmartin — COMMENTED** [review]
Mostly nitpicks that you're free to ignore, I didn't see find any bugs. That said, I was only looking at the diff and not the original code you were fixing, so if there were other issues in code you didn't change, I wouldn't have seen them.

File: app/views/houses/edit.html.slim (hunk 4 above)
Comment (nsixcancode) [reply app/views/houses/edit.html.slim:18]: readded to controller

File: app/views/submissions/_form.html.slim (hunk 5 above)
Comment (nsixcancode) [reply app/views/submissions/_form.html.slim:213]: removed pluralization.

**Pull request merged**